doc/releases/changelog-dev.md

```diff
@@ -292,6 +344,10 @@
   when converting a `QuantumCircuit` using `qml.from_qiskit`.
   [(#5168)](https://github.com/PennyLaneAI/pennylane/pull/5168)
 
+* Added new error tracking and propagation functionality. 
```

Comment:
not so obvious what this one is about

Reply:
I think this one is ok - it'll be a major feature to advertise in 0.36 but we don't want to focus on it too much in 0.35.